src/compat/libc/stdlib/mbtowc.c

```diff
@@ -0,0 +1,30 @@
+/**
+ * @file
+ * @brief mbtowc function.
+ *
+ * @see stdlib.h
+ *
+ * @date 01.11.18
+ * @author Chubukov Filipp
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <defines/size_t.h>
+#include <defines/wchar_t.h>
+
+int mbtowc(wchar_t *out, const char *in, size_t n) {
+	int i = 0;
+	if (in == NULL) {
```

Comment:
You also need to check that `out != NULL`, because you use this pointer.

Reply:
added